demos/nodejs/hapi/server.js

```diff
@@ -0,0 +1,127 @@
+const Hapi = require('@hapi/hapi');
+
+const config = require('./config');
+const lrObj = require('loginradius-sdk')(config);
+
+const path = require('path');
+
+const VIEW_PATH = path.join(__dirname, '../../../theme');
+
+const init = async () => {
+
+    const server = Hapi.server({
+        port: 3000,
+        host: 'localhost',
+        routes: {
+            files: {
+                relativeTo: VIEW_PATH
+            }
+        }
+    });
+
+    await server.register(require('@hapi/inert'));
+    server.route({

```

Comment:
@karmanya007 It would be better if you can add comments for routes.

Reply:
Okay.